src/main/scala/diplomacy/Nodes.scala

```diff
@@ -474,17 +1233,34 @@ sealed abstract class MixedNode[DI, UI, EI, BI <: Data, DO, UO, EO, BO <: Data](
   }
 
   private var bundlesSafeNow = false
-  // Accessors to the result of negotiation to be used in LazyModuleImp:
+
+  /** Gather Bundle and edge parameters of outward ports.
+    *
+    *  Accessors to the result of negotiation to be used within [[LazyModuleImp]] Code
+    */
   def out: Seq[(BO, EO)] = {
     require(bundlesSafeNow, s"$name.out should only be called from the context of ${scope.get.name}'s LazyModuleImp, but the current scope is ${LazyModule.scope}.")
     bundleOut zip edgesOut
   }
+
+  /** Gather Bundle and edge parameters of inward ports.
+    *
+    *  Accessors to the result of negotiation to be used within [[LazyModuleImp]] Code
+    */
   def in: Seq[(BI, EI)] = {
     require(bundlesSafeNow, s"$name.in should only be called from the context of ${scope.get.name}'s LazyModuleImp, but the current scope is ${LazyModule.scope}.")
     bundleIn zip edgesIn
   }
 
-  protected[diplomacy] def instantiate() = {
+  /* If this node does not need monitoring, set this value to true. */
+  protected val identity = false
```

Comment:
I think this got replaced by `circuitIdentity` recently and should be removed entirely.